### ISSUE TITLE
Resource not deleted when change Principal Id of a SSO Assignment Group

### ISSUE THREAD

**Issue by eduardomourar** [org-formation/aws-resource-providers#65]
Whenever I modify the property `PrincipalId`, the assignment is created for the the new one, but the old one is not fully deleted.

This is happening in`Community::SSO::AssignmentGroup` resource type v0.3.1.

**Comment by OlafConijn** [comment]
might be an idea to not allow changing the PrincipalId?

**Comment by eduardomourar** [comment]
yes, i think it can be in the `createOnlyProperties` because that will trigger creating a new resource whenever you modify PrincipalId
